app/models/author.rb

```diff
@@ -0,0 +1,4 @@
+class Author < ApplicationRecord
+    has_many :posts
+    has_many :comments
```

Comment:
The author and comments would not be having any associations. Please remove it.

Reply:
resolved